.github/workflows/format-prettier.yml

```diff
@@ -0,0 +1,52 @@
+name: Format with Prettier
+
+on:
+  push:
+    branches: [ 'main' ]
+    paths:
+      - '**.md'
+      - '**.yml'
+      - '**.yaml'
+  pull_request:
+    branches: [ 'main' ]
+    paths:
+      - '**.md'
+      - '**.yml'
+      - '**.yaml'
+  workflow_dispatch:
+
+jobs:
+  format-markdown:
+    runs-on: ubuntu-latest
```

Comment:
One potential issue with the current workflow is that when github actions pushes the automated update, it retriggers the CI pipeline, which is unnecessary and may need to infinite loops.

According to ChatGPT, we can add the following line to prevent CI pipeline being triggered by itself:
`if: "!contains(github.event.head_commit.message, 'chore: format files with Prettier')"`

Reply:
I'll confirm this.